scripts/python/tao_build.py

```diff
@@ -402,6 +403,9 @@ def test_tao_compiler(root, args):
     TARGET_DISC_CUDA_SOURCE_TESTS = [
         "//tensorflow/compiler/mlir/disc/tools/disc-source-emitter/tests/..."
     ]
+    TARGET_DISC_TRANSFORM_DIALECT_TESTS = [
```

Comment:
This file changes too much, add @wyzero for review.

Reply:
Maybe we need to rebase to the main branch first? this part of code seems to be from the main branch.